roles/ceph-osd/tasks/main.yml

```diff
@@ -1,4 +1,15 @@
 ---
+- name: delete default 'rbd' pool
+  command: ceph osd pool delete rbd rbd --yes-i-really-really-mean-it
+  register: poolout
+  changed_when: poolout.stdout | search('removed')
+  delegate_to: "{{ groups['ceph_monitors'][0] }}"
+  when: inventory_hostname == groups['ceph_osds'][0] 
+
+- include: pool_names.yml
+  delegate_to: "{{ groups['ceph_monitors'][0] }}"
+  when: inventory_hostname == groups['ceph_osds'][0] 
```

Comment:
in `site.yml` we play `ceph-osd` role with `serial: 1`. As I see, `run_once: true` not really run once with `searial:1 `, it run against every host